package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1399,7 +1409,8 @@ def asphericity(group, pbc=False, unwrap=None, compound='group'):
         ('asphericity', asphericity))
 
     @warn_if_not_unique
-    def principal_axes(group, pbc=False):
+    @_pbc_to_wrap
+    def principal_axes(group, wrap=False):
```

Comment:
versionchanged and deprecation note missing

Reply:
Should this function be able to use `unwrap` and `compound`? If so, raise issue.